app.js

```diff
@@ -0,0 +1,185 @@
+var timeLabelsArray = ['10AM: ','11AM: ','12PM: ', '1PM: ', '2PM: ', '3PM: ', '4PM: ', '5PM: ', 'SUM: ']
+var pikePlace = {
+  min: 17,
+  max: 88,
+  avg: 5.2,
+  randomOutput: function(){
+    return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
+  },
+  dailyRandomOutput: function(){
+    var cookiesArray = [];
+    for(i = 0; i < 8; i++){
```

Comment:
This isn't the most scalable way to handle this. What if you decided you wanted to keep your stores open for 3 hours later, and added those extra hours to your hours array?

Your final index in your hours array is 'SUM:', but you could perhaps consider refactoring this for loop to handle your hours changing, but still be able to deal with the SUM: